Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
I more or less just didn't get why you didn't merge it into the same commit, considering your first commit makes direct use of it, (excluding indirect includes, it'd be like committing a compile error and then fixing it in the same change set) and it seems kind of odd to have a commit for something fairly remediable 'as you go', since it's not really significant, which kinda seems line-noisy history-wise?

idk, I don't really mind either way though. Was just trying to understand the thought process.